packages/components/src/visually-hidden/component.js

```diff
@@ -1,8 +1,16 @@
 /**
  * Internal dependencies
  */
-import { createComponent } from '../ui/utils';
-import { useVisuallyHidden } from './hook';
+import { useContextSystem, contextConnect } from '../ui/context';
+import { VisuallyHidden as VisuallyHiddenWrapper } from './styles';
+
+/**
+ * @param {import('../ui/context').PolymorphicComponentProps<{ children: import('react').ReactNode }, 'div'>} props
+ */
+function VisuallyHidden( props ) {
+	const contextProps = useContextSystem( props, 'VisuallyHidden' );
+	return <VisuallyHiddenWrapper { ...contextProps } />;
```

Comment:
Should we call this component `VisuallyHiddenView` to match the naming convention used in other PRs?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/31902/commits/35892e1d8a4f4ecf9a249a2f8ed2134163ef426b